public/res/default.json

```diff
@@ -19,7 +19,9 @@
   },
   "home-page": {
     "title": "SciGateway",
+    "logo-label": "SciGateway Logo",
     "how-label": "How it works...",
+    "background-label": "NERC JASMIN supercomputer based at STFC’s Rutherford Appleton Laboratory",
```

Comment:
I remember we agreed that this would be good alt text for the background image behind the SciGateway logo but it is not being used on the image as it stands. Personally I am fine with this, because it could conflict with the alt text on the logo itself but if we choose to leave it off the image, we should remove this from `default.json`

Reply:
Yes, I thought this too, but left it in as it might be useful in the future if needed. I have removed it now and committed as it is cleaner.